core/scheduler/service/executor_input_compiler.go

```diff
@@ -136,6 +136,15 @@ func (i InputCompiler) Compile(ctx context.Context, job *scheduler.JobWithDetail
 		"job_name":  job.Job.Name.String(),
 		"job_id":    job.Job.ID.String(),
 	}
+
+	if job.JobMetadata != nil {
```

Comment:
I believe hook should also need job labeling as well https://github.com/goto/optimus/pull/187/files#diff-a858557c85ba3d339a5a2c95182e636538521faa9808b73628427002e0dcf9ebR180, eg. transporter